.travis.yml

```diff
@@ -12,16 +11,19 @@ php:
 matrix:
   allow_failures:
     - php: hhvm
+  include:
+    - php: 5.4
+      env: COMPOSER_FLAGS="--prefer-lowest"
 
 install:
   - eval `ssh-agent -s`
   - test/travis/setup-secure-shell.sh
   - composer self-update
-  - composer install --no-interaction --prefer-source
```

Comment:
I'd drop it since it's EOL anyways.

So. - Since the builds pass and the removal of php 5.4 support is not subject to the pull request, do you think we can get this in?